analyses/telomerase-activities-prediction/03-RSEM-Counts-Stranded.r

```diff
@@ -0,0 +1,27 @@
+library('pracma')
+library('dplyr')
+library('ggpubr')
+stringsAsFactors=FALSE
+library(circlize)
+library(stringr)
+library('devtools')
+library(EXTEND)
+
+
+################################################# Running Analysis on Gene Counts #######################################################################################
```

Comment:
Hi Dr. Taroni...
Thanks for the detailed description...
So according to your first pull request requirement we have created the shell script, 01-run-EXTEND.R  script and have done the modifications in the docker file. Now we wonder if we need to create a new branch and pick up the commit from previous branch to submit the new pull request.

Reply:
Hi @NNoureen - there are a few options. 

* You can create a new branch, add the you changes describe to the new branch, and file a new pull request. If you have modified the Dockerfile, you don't need to use `git cherry-pick` to get the changes in the commit where the Dockerfile is altered and you're good to skip that step.
* You can make the changes you described to this branch `Telomerase-Activity-Prediction`. I thought it might be handy to keep this open and move over files as necessary to new branches (the later scripts for example). 

Does this answer your question? Thanks!